packages/simple-exchange-frontend/src/store/reducer.js

```diff
@@ -20,30 +20,42 @@ function randomName(max) {
   return result.join('');
 }
 
-function createFakeOrderBook(count) {
-  const result = [];
-  for (let i = 0; i < count; i += 1) {
-    result.push({
-      id: i,
-      side: randomArrayItem(['Buy', 'Sell']),
-      size: randomInteger(1000),
-      filled: randomInteger(1000),
-      my_size: randomBoolean() ? randomInteger(1000) : undefined,
-    });
-  }
+// function createFakeOrderBook(count) {
+//   const result = [];
+//   for (let i = 0; i < count; i += 1) {
+//     result.push({
+//       id: i,
+//       side: randomArrayItem(['Buy', 'Sell']),
+//       size: randomInteger(1000),
+//       filled: randomInteger(1000),
+//       my_size: randomBoolean() ? randomInteger(1000) : undefined,
+//     });
+//   }
+//   return result;
+// }
+
+function createFakeSide(side) {
+  const allegedName = side ? 'moola' : 'simoleans';
+  const result = {
+    label: { assay: {}, allegedName },
+    extent: randomInteger(1000),
+  };
   return result;
 }
 
-function createFakeOrderHistory(count) {
-  const result = [];
-  for (let i = 0; i < count; i += 1) {
-    result.push({
-      id: i,
-      side: randomArrayItem(['Buy', 'Sell']),
-      size: randomInteger(1000),
-      filled: randomInteger(1000),
-      status: randomArrayItem(['Open', 'Filled']),
-    });
+function createFakeOrder() {
+  const order = randomBoolean();
```

Comment:
In the [sample update I provided](https://github.com/Agoric/agoric-sdk/issues/541#issuecomment-588577553), one set of orders is labelled 'buys', and the other is labelled 'sells'. Both have 'offers' and 'wants'. They buys all have wants that are in moola, and offers that are in simoleans. The sells are all opposite: the offers are in moola, and wants in simoleans.

Reply:
@Chris-Hibbert, see my comment here:

https://github.com/Agoric/agoric-sdk/issues/611

The buy and sell depend on how the exchange is set "moola/simoleans" or "simoleans/moola". We have to define that otherwise we risk to have inconsistencies.